app/forms/new_article_form.rb

```diff
@@ -0,0 +1,15 @@
+class NewArticleForm
+  include ActiveModel::Model
+
+    attr_accessor(
+        :title,
+        :text
+    )
+
+    validates (
+        :title,
+        :text,
+        presence: true
+    )
+
+end
```

Comment:
the alignment is still WRONG
![image](https://user-images.githubusercontent.com/11631898/79503731-5625d580-803a-11ea-9297-082eed82ccfe.png)

can't you see line 2, has 2 spaces, line 4 has 4 spaces (BUT it should have only 2)